packages/core/admin/admin/src/pages/ProfilePage/tests/index.test.js

```diff
@@ -22,27 +25,30 @@ jest.mock('@strapi/helper-plugin', () => ({
   useOverlayBlocker: jest.fn(() => ({ lockApp: jest.fn, unlockApp: jest.fn() })),
 }));
 
-const client = new QueryClient({
-  defaultOptions: {
-    queries: {
-      retry: false,
+
+  const client = new QueryClient({
+    defaultOptions: {
+      queries: {
+        retry: false,
+      },
     },
-  },
-});
+  });
```

Comment:
If I do that I have a weird behaviour in my test, basically it stays in Loading phase without content so I am not able to test the fields

Reply:
That seems odd and sounds like a side-effect. Imo it should be possible to move client into wrapper, because that is what we do elsewhere :)

Reply:
Well, not really, in the majority of the scenarios it is declared outside

Reply:
Yes - but it makes sense to move it in there and we are slowly moving them, because there is no need to have the query client as a global in the tests.